src/main/features/response/routes/timeline.ts

```diff
@@ -41,7 +41,15 @@ export default express.Router()
     page.uri,
     async (req: express.Request, res: express.Response, next: express.NextFunction) => {
       const draft: Draft<ResponseDraft> = res.locals.responseDraft
-      renderView(new Form(draft.document.timeline), res)
+      let timeline
+
+      if (draft.document.isResponsePartiallyAdmitted()) {
+        timeline = draft.document.partialAdmission.timeline
```

Comment:
why does this go under partial admission and not the timeline?

Reply:
According to @geek-soft we want to keep everything related to `PartialAdmission` (or `FullAdmission`) in separate nested object.

Next step will be extracting timeline and evidence to reusable utils/partials/services so that we will be able to create it easily with different paths. Something similar to what we have here:
SealedClaimPdfGenerator.requestHandler